### PR TITLE
For #1370: Added text to prove that NPE does not happen.

### PR DESCRIPTION
**Pull request by paulodamaso**
For #1370: Added test to prove that NPE does not happen.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1433](https://github.com/jcabi/jcabi-github/issues/1433) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1433](https://github.com/jcabi/jcabi-github/issues/1433) is assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C9R53K5JA/2431953380); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by paulodamaso** [comment]
@llorllale ping

**Comment by paulodamaso** [comment]
@llorllale ping

File: src/test/java/com/jcabi/github/mock/MkGithubTest.java [hunk 1]
```diff
@@ -179,4 +180,19 @@ public Void call() throws Exception {
...
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void canRetrieveUsers() throws Exception {
+        MatcherAssert.assertThat(
+            "Retrieved inexistent user",
+            new User.Smart(
+                new MkGithub().users().get("other")
+            ).exists(),
+            new IsEqual<>(true)
```
Comment (amihaiemil) [inline src/test/java/com/jcabi/github/mock/MkGithubTest.java]: @paulodamaso but, according to the ticket's [description](https://github.com/jcabi/jcabi-github/issues/1370#issue-306858750) this should be ``false`` since the user doesn't actually exist

File: pom.xml [hunk 2]
```diff
@@ -204,6 +204,14 @@ OF THE POSSIBILITY OF SUCH DAMAGE.
...
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.7</source>
+          <target>1.7</target>
+        </configuration>
+      </plugin>
```
Comment (amihaiemil) [inline pom.xml]: @paulodamaso Why did you add this? :D

File: src/test/java/com/jcabi/github/mock/MkGithubTest.java (hunk 1 above)
Comment (paulodamaso) [reply src/test/java/com/jcabi/github/mock/MkGithubTest.java]: @amihaiemil Sorry, I didn't noticed that. The problems: `MkUsers().get("")` is creating a new user even when we does not have this user in "database". I've added a method `add(login)`  to create the user and removed creating part of `get(login)`.

File: pom.xml (hunk 2 above)
Comment (paulodamaso) [reply pom.xml]: @amihaiemil My IDE was complaining about it, but it was some local coniguration.

**Comment by paulodamaso** [comment]
@amihaiemil Answered your comments, please take a look.

**Comment by amihaiemil** [comment]
@rultor good to merge

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31):

**Comment by 0crat** [comment]
The job [#1433](https://github.com/jcabi/jcabi-github/issues/1433) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)
